paths/job_locales/index.yaml

```diff
@@ -11,6 +11,7 @@ parameters:
 - "$ref": "../../parameters.yaml#/page"
 - "$ref": "../../parameters.yaml#/per_page"
 - "$ref": "../../parameters.yaml#/branch"
+- "$ref": "../../parameters.yaml#/include_annotations"
```

Comment:
I am not sure if we should document this public. But also no strong opinion against it.